app/templates/partials/summary/answer-summary.html

```diff
@@ -31,4 +33,8 @@ <h2 class="summary__title neptune">{{ content.summary.label }}</h2>
     {%- endfor -%}
   </div>
   {%- endfor -%}
+  <a class="page__previous page__previous--bottom u-mb-l u-dib"
+     {{helpers.track('click', 'Navigation', 'Add Person link click', bottom)}}
```

Comment:
Since the idea to make this generic so it can be reused, can we not say `Person`?

Reply:
This wouldn't be the generic solution going forward. This is an LMS compromise for the time being so I didn't spend too much time worrying about nomenclature.